.travis.yml

```diff
@@ -47,6 +47,21 @@ matrix:
            PROJECT_URL=https://github.com/joblib/joblib.git
       python: 3.7
       if: commit_message =~ /(\[ci downstream\]|\[ci joblib\])/
+    - os: linux
+      env: PROJECT=ray
+      python: 3.7
+      if: commit_message =~ /(\[ci downstream\]|\[ci ray\])/
+      install:
```

Comment:
The Ray Travis build should probably also include its own `before_script:` (which is just empty) to avoid running the default one.

Reply:
Actually maybe not, since we do want to run `install_coverage_subprocess_pth.py` I think.